Streetcode/DbUpdate/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using DbUp;
+using Microsoft.Extensions.Configuration;
+
+public class Program
+{
+    static int Main(string[] args)
+    {
+        string seedingPath = Directory.GetCurrentDirectory() + @"\Streetcode.DAL\Persistence\ScriptsMigration\";
```

Comment:
It's better to use Path.Combine for this purpose

Reply:
got it!